Rakefile.dist

```diff
@@ -1,4 +1,5 @@
 require 'rake/clean'
+load File.expand_path('../lib/tasks/pkg.rake', __FILE__)
```

Comment:
I wonder if `require_relative` would work here…

Reply:
That sadly doesn't work. I suspect because it doesn't have a `.rb` extension but a `.rake`. My ruby knowledge is lacking.